components/Chips/examples/ChipsShapingExampleViewController.m

```diff
@@ -0,0 +1,113 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ChipsExamplesSupplemental.h"
+
+#import "MaterialChips.h"
+#import "MaterialSlider.h"
+#import "MaterialShapes.h"
+#import "MaterialShapeLibrary.h"
+
+@implementation ChipsShapingExampleViewController {
+  MDCChipView *_chipView;
+  MDCSlider *_cornerSlider;
+  MDCRectangleShapeGenerator *_rectangleShapeGenerator;
+  UISegmentedControl *_cornerStyleControl;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  self.view.backgroundColor = [UIColor whiteColor];
+
+  _rectangleShapeGenerator = [[MDCRectangleShapeGenerator alloc] init];
+
+  _chipView = [[MDCChipView alloc] init];
+  _chipView.titleLabel.text = @"Material";
+  _chipView.imageView.image = [self faceImage];
+  _chipView.accessoryView = [self deleteButton];
+  _chipView.imagePadding = UIEdgeInsetsMake(0, 10.f, 0, 0);
+  _chipView.accessoryPadding = UIEdgeInsetsMake(0, 0, 0, 10.f);
+  [self.view addSubview:_chipView];
+
+  CGSize chipSize = [_chipView sizeThatFits:self.view.bounds.size];
+  _chipView.frame = CGRectMake(20, 20, chipSize.width + 20, chipSize.height + 20);
```

Comment:
Any reason for the 20's?

Reply:
For better visual spacing and sizing to demonstrate the example better